src/upf/src/upf.tcl

```diff
@@ -301,5 +301,52 @@ proc set_domain_area { args } {
   upf::set_domain_area_cmd $domain_name $llx $lly $urx $ury
 }
 
+# Specify which power-switch model is to be used for the implementation of the corresponding switch
+# instance
+#
+#
+# Argument list: 
+#  - switch_name_list []: A list of switches (as defined by create_power_switch) to map.
+#  - lib_cells [] : A list of library cells.
+#  - port_map {{mapped_model_port switch_port_or_supply_net_ref} *} : mapped_model_ port is a port on the model being mapped.
+#                                                                     switch_ port_or_supply_net_ref indicates a supply or logic port on a
+#                                                                     switch: an input supply port, output supply port, control port, or
+#                                                                     acknowledge port; or it references a supply net from a supply set
+#                                                                     associated with the switch.
```

Comment:
@luarss 
FYI, not sure if we will merge the docs PR on upf before or after this, but this will need to be included.

Reply:
@ahmadelrouby we have merged the UPF docs PR, could you update the README to include the new command?

Reply:
I added it to the UPF readme file

Reply:
Should we remove the comments or replace them with a pointer to the README file? I am worried that in the future if one changes the code it will forget to change both places where the arguments are defined (code and docs).

Reply:
I get that, but all these methods are mirrors to the spec of UPF, but I'll make sure I updated in both places from now on.